compose.yaml

```diff
@@ -35,31 +40,51 @@ services:
     restart: on-failure
     ports:
       - "${CSAF_KEYCLOAK_DATABASE_PORT}:5432"
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U $${CSAF_KEYCLOAK_DATABASE_USER}"]
+      interval: 10s
+      timeout: 10s
+      retries: 10
     networks:
       default:
         aliases:
           - "keycloak-db.csaf.internal"
 
   keycloak:
-    image: quay.io/keycloak/keycloak:20.0
+    image: quay.io/keycloak/keycloak:26.2
     hostname: keycloak.csaf.internal
     #container_name: keycloak
     env_file: .env
     environment:
       # https://www.keycloak.org/server/all-config
       KC_HEALTH_ENABLED: "true"
-      KC_METRICS_ENABLED: "true"
       KC_DB: postgres
+      KC_DB_SCHEMA: public
+
       KC_DB_URL_HOST: keycloak-db.csaf.internal
       KC_DB_URL_PORT: 5432
       KC_DB_URL_DATABASE: ${CSAF_KEYCLOAK_DATABASE_NAME}
+      
       KC_DB_USERNAME: ${CSAF_KEYCLOAK_DATABASE_USER}
       KC_DB_PASSWORD: ${CSAF_KEYCLOAK_DATABASE_PASSWORD}
-      KC_DB_SCHEMA: public
+      
       KEYCLOAK_ADMIN: ${CSAF_KEYCLOAK_ADMIN_USER}
       KEYCLOAK_ADMIN_PASSWORD: ${CSAF_KEYCLOAK_ADMIN_PASSWORD}
+      
+      KEYCLOAK_EXTRA_ARGS: --import-realm
+    volumes:
+      - ./keycloak:/opt/bitnami/keycloak/data/import
```

Comment:
I had to change this entry to get the import work

    volumes:
      - ./keycloak/csaf-realm.json:/opt/keycloak/data/import/realm.json:ro

Reply:
Moved initial setup to script. This should solve the problem.